stored_requests/backends/file_fetcher/fetcher.go

```diff
@@ -35,8 +35,10 @@ func (fetcher *eagerFetcher) FetchRequests(ctx context.Context, requestIDs []str
 	return storedRequests, storedImpressions, errs
 }
 
+// Fetch Responses - Implements the interface to read the stored response information from the fetcher's FileSystem, the directory name is "stored_responses"
 func (fetcher *eagerFetcher) FetchResponses(ctx context.Context, ids []string) (data map[string]json.RawMessage, errs []error) {
-	return nil, nil
+	storedResponses := fetcher.FileSystem.Directories["stored_responses"].Files
```

Comment:
Pre Gus suggestion, please use "data", not "storedResponses" https://github.com/prebid/prebid-server/pull/3326#discussion_r1544811287

Reply:
@VeronikaSolovei9 @guscarreon 
have updated the code implementation